ProcessQueue/__init__.py

```diff
@@ -476,3 +479,71 @@ def main(msg: func.QueueMessage):
             field_ids = []
             field_names = []
             data = []
+    if parent_sr_id is not "None":
+        # Grab Parent Scan Report
+        parent_scan_report = requests.get(
+                    url=api_url
+                    + "scanreporttablesfilter/?scan_report="
+                    + str(parent_sr_id),

```

Comment:
pedantic comment - you don't need to fix :P !

it's more in line with pep-8 style to do this as one line, and maybe better using f-strings. i.e.
```
url=f"{api_url}scanreporttablesfilter/?scan_report={parent_sr_id}"
```

Reply:
For my education, is there any difference between the `scanreporttablesfilter/?scan_report=10` and `scanreporttables/10`? Either in functionality or performance?

Reply:
@shahzadmumtaz22  correct me if i'm wrong, but the `scanreporttablesfilter` could be better to use (performance wise) as you're able to also specify which `fields` you need returning. The `scanreporttables` api will return everything (the full serialised model) for that report.

E.g. you can do (?) something like:
```
scanreporttablesfilter/?scan_report=10?fields=id,name
```
which just returns a json with the `id` and `name`, rather than all the junk about date_created and other fields you don't need etc. 

No idea if there's any difference if they're used as in @spco's comment.

Reply:
Ok, so the choice of which is "right" will depend on what we use from the response. Ta.

Reply:
@vpnu 
![image](https://user-images.githubusercontent.com/53081016/130676412-4f2e9595-5b59-49f7-8489-a08ce0b9b51a.png)

http://localhost:8080/api/scanreporttablesfilter/?scan_report=1 (this returns details of all the tables for a scanreport that has an id=1 (this is scan report id, a field that is in the scan_report_table entries to link back to scan_report). 

http://http://localhost:8080/api/scanreporttables/10 (this returns a table that has an id value=10 in the scanreporttable contents). In an api when you use **/api/tablename/primarykeyvalue**, a primarykeyvalue is usually an id for a given table whereas when you use **filter/?filter_field=[]** whereas filter_field is usually a field other than the primary key field. 

@calmacx You are right we can specify returning fields (as we did earlier for couple of endpoints) to make it cleaner when using it for any purpose. 

@calmacx , @spco and @vpnu  I have update all serializers to allow users to specify returning fields for all API endpoints (except for OMOP tables and if required the same can be replicated to those as well). 
![image](https://user-images.githubusercontent.com/53081016/130680835-079e61eb-81cf-4ecb-b0c9-00eef48d7c8c.png)